web/turkey/survey/static/survey/js/auditors/keypresses_total.js

```diff
@@ -0,0 +1,25 @@
+var AuditorKeypressesTotal = {
+    keypresses_total: 0,
+    log_keypress_event: function (e) {
+        // e.stopPropagation();
+        this.keypresses_total += 1;
+    },
+    submit_callable: function () {
+        return {
+            'count': this.keypresses_total
+        };
+    }
+};
+
+var auditor_keypresses_total = Object.create(AuditorKeypressesTotal);
+
+$(document).ready(function() {
+    $(document).keydown(auditor_keypresses_total
```

Comment:
I handled this in the original auditor.js, e.stopPropagation I think fixes this. But I think I used a different jQuery handler, so I haven't tested it yet. When I originally coded this out, holding down a key only counted for one keypress.